### DISCUSSION TITLE
Implement oauth for third party users to allow for app creation

### DISCUSSION THREAD

**Discussion by cmclark00** [sigma67/ytmusicapi#537] [Ideas]
I am working on a webapp that would be so much better if I could use this api vs the youtube data api. If an oauth 2.0 flow could be implemented for users visiting said app/webapp and then having oauth.json log the individual users information this would be perfect for my use case and would open it up for a lot of other uses as well.

**Comment by lawsonj512** [comment]
Brilliant idea, this would be ideal for me too!

**Comment by sigma67** [comment]
This is already possible and has been done by others using the APIs currently implemented afaik
